mapper/mapper.go

```diff
@@ -122,9 +124,12 @@ func CreateFilterOverview(dimensions []filter.ModelDimension, filter filter.Mode
 func CreateListSelectorPage(name string, selectedValues []filter.DimensionOption, allValues dataset.Options, filter filter.Model, dst dataset.Model, datasetID, releaseDate string) listSelector.Page {
 	var p listSelector.Page
 
+	log.Debug("mapping api response models to list selector page model", nil)
```

Comment:
same as above, some data might be useful - dataset ID and list name perhaps?

generally all the following log lines around 'mapping models' - some values will be useful in each